packages/account/src/Components/poi/poi-confirm-with-example-form-container/poi-confirm-with-example-form-container.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Form, Formik, FormikHelpers } from 'formik';
+import { GetSettings } from '@deriv/api-types';
+import { Checkbox, Loading } from '@deriv/components';
+import { filterObjProperties, toMoment, validLength, validName, WS } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import FormBody from 'Components/form-body';
+import LoadErrorMessage from 'Components/load-error-message';
+import PersonalDetailsForm from 'Components/forms/personal-details-form';
+import { validate } from 'Helpers/utils';
+
+type TValues = { [p: string]: string };
+
+type TRestState = {
+    api_error: string;
+    show_form: boolean;
+    errors?: boolean;
+    form_initial_values: TValues;
+    changeable_fields: string[];
+};
+
+type TPoiConfirmWithExampleFormContainer = {
+    account_settings: GetSettings;
+    getChangeableFields: () => string[];
+    onFormConfirm?: () => void;
+};
+
+const PoiConfirmWithExampleFormContainer = ({
+    account_settings,
+    getChangeableFields,
+    onFormConfirm,
+}: TPoiConfirmWithExampleFormContainer) => {
+    const [is_loading, setIsLoading] = React.useState(true);
+    const [checked, setChecked] = React.useState(false);
+    const [rest_state, setRestState] = React.useState<TRestState>({
+        show_form: true,
+        form_initial_values: { first_name: '', last_name: '', date_of_birth: '' },
+        changeable_fields: [],
+        api_error: '',
+    });
+
+    React.useEffect(() => {
+        initializeFormValues();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
if I add all what is asked by eslint here, I will get circular dependence

Reply:
@yauheni-deriv That's a red flag that something is wrong with the implementation, Whenever you find yourself in a case where adding the right dependencies to the useEffect dependency array makes unexpected behavior that means you probably need to change your implementation as it's buggy.

So for this case, `initializeFormValues` is defined in the component body, That means by each rerender a new `initializeFormValues` function will be created, Which means a different thing so useEffect will be called again. 

You can wrap the function with `useCallback` hook to avoid creating a new function on each rerender, Or in your case here, You can just move the function declaration itself in the useEffect hook and that fixes the whole thing.